doc/reference/storage/fcb/fcb.rst

```diff
@@ -0,0 +1,77 @@
+.. _fcb:
+
+Flash Circular Buffer (FCB)
+###########################
+
+Flash circular buffer provides an abstration through which you can treat
+flash like a FIFO. You append entries to the end, and read data from the
+beginning.
+
+Description
+***********
+
+Elements in the flash contain the length of the element, the data within
```

Comment:
Are ``elements`` the same as ``entries``?  Can we use the same term everwhere in this doc?

Reply:
Yes they are entries. I switch to using entry elsewhere.